integration-test-groups/foundation/scheduler/src/main/java/org/apache/camel/quarkus/component/scheduler/it/SchedulerRoute.java

```diff
@@ -30,11 +31,24 @@ public class SchedulerRoute extends RouteBuilder {
     @Inject
     @Named("schedulerCounter")
     AtomicInteger schedulerCounter;
+    @Inject
+    CamelContext context;
 
     @Override
     public void configure() throws Exception {
-        from("scheduler:start?initialDelay=1")
+        from("scheduler:withInitialDelay?initialDelay=1")
                 .process(e -> schedulerCounter.incrementAndGet());
 
+        from("scheduler:withDelay?delay=1000")
+                .to("bean:myCounter?method=increment");
+
+        from("scheduler:useFixedDelay?initialDelay=2000&useFixedDelay=true")
+                .to("bean:myCounter?method=increment");
```

Comment:
I'd vote for using shorter delays, to save some testing time. 200ms should be fine. At the same time, we should not check using `==` in the tests, only `<`, for the case that the test is too slow to observe the exact value.